src/ontology/config/properties.txt

```diff
@@ -26,3 +26,4 @@ rdfs:label
 rdfs:seeAlso
 owl:deprecated
 http://purl.org/dc/terms/description
+http://purl.obolibrary.org/obo/mondo#ABBREVIATION
```

Comment:
Remember `http://purl.obolibrary.org/obo/OMO_0003012` is needed.

Reply:
No worries. That is included in:
- #93